packages/zoltan2/test/sphynx/Sphynx_Research_Driver.cpp

```diff
@@ -492,14 +386,14 @@ int main(int narg, char *arg[])
         params->set("pulp_vert_imbalance", 1.01);
       }
 
-      using problem_type = Zoltan2::SphynxProblem<adapter_type>;
+      using problem_type = Zoltan2::PartitioningProblem<adapter_type>;
       Teuchos::RCP<problem_type> problem;
       pComm->barrier();
       {
-        Teuchos::TimeMonitor t1(*Teuchos::TimeMonitor::getNewTimer("Partitioning::All"));
+        Teuchos::TimeMonitor t(*Teuchos::TimeMonitor::getNewTimer("Partitioning::All"));
         {
           Teuchos::TimeMonitor t2(*Teuchos::TimeMonitor::getNewTimer("Partitioning::Problem"));
-          problem = Teuchos::rcp(new problem_type(adapter.getRawPtr(), params.getRawPtr(), sphynxParams, Tpetra::getDefaultComm()));
+          problem = Teuchos::rcp(new problem_type(adapter.getRawPtr(), params.getRawPtr(), Tpetra::getDefaultComm()));
```

Comment:
So where are the sphynxParams getting passed in?

Reply:
This isn't using Sphynx, this particular call is using parmetis or pulp. Do I still add in the parameters in that case?